packages/neos-ui-editors/src/Editors/Image/Components/PreviewScreen/index.js

```diff
@@ -54,7 +54,7 @@ export default class PreviewScreen extends PureComponent {
                             className={(thumbnail ? style.cropArea__image : style['cropArea__image--placeholder'])}
                             src={thumbnail ? thumbnail.uri : '/_Resources/Static/Packages/Neos.Neos/Images/dummy-image.svg'}
                             style={thumbnail ? thumbnail.styles.thumbnail : {}}
-                            role="presentation"
+                            alt="thumbnail"
```

Comment:
This alt attribute would be used to display a tooltip on hover, and this tooltip would carry 0 relevant information, and for non-english speaking editors that'd be even annoying. I propose to use a `propertyName` for the alt attribute, at least it's slightly relevant to the information provided.

Reply:
Thanks for that suggestion. Fixed.